Firestore/tests/Unit/SnapshotTraitTest.php

```diff
@@ -45,6 +45,7 @@ class SnapshotTraitTest extends TestCase
     private $connection;
     private $mapper;
     private $impl;
+    private $valueMapper;
```

Comment:
Same for this `valueMapper`.

Reply:
PHP automatically defines properties which are not defined explicitly. Only from PHP8.2 we see deprecation notices, [ref](https://www.php.net/manual/en/class.allowdynamicproperties.php#example-1231).